### DISCUSSION TITLE
GitHub packages naming collision

### DISCUSSION THREAD

**Discussion by mfridman** [community/community#26061] [Actions]
<p>Hey all,</p>
<p>We have 2 repos within the same org using GitHub packages.</p>
<p>Since it’s mandatory to prefix the images with the repo identifier, why is it not possible to have 2 images with the same name?</p>
<p>E.g., owner/repo1/alpine and owner/repo2/alpine</p>
<p>This was the error:</p>
<pre><code class="lang-auto">blob upload invalid: Package "alpine" is already associated with another repository.
</code></pre>

**Comment by thomasjo** [comment, marked as answer]
<p>Packages are defined at the repository level but associated with a user/organization. Because of this, the package name (and version) is reserved at the user/organization level and can <em>never</em> be re-used.</p>
<p>This is something that should perhaps be a little bit better documented. It is briefly mentioned in the third paragraph in <a href="https://docs.github.com/en/packages/publishing-and-managing-packages/publishing-a-package#about-published-packages" rel="nofollow noopener">“about published packages”</a>.</p>
